Pipfile

```diff
@@ -6,6 +6,7 @@ name = "pypi"
 [packages]
 pygame = "*"
 requests = "*"
+pyyaml = "*"
```

Comment:
Since we're using a yaml library, we should define ***E V E R Y T H I N G*** as yamls.